### PR TITLE
adding flash messages and refactored code thanks to chris

### PR DESCRIPTION
**Pull request by Hemesh-Unka**

### CODE REVIEW

**Review by Reena-Koudi — COMMENTED** [review]
Having the if condition in the alert message would be nicer!

<% if notice %>
    <p class="alert alert-success"><%= notice %></p>
    <% end %>
    <% if alert %>
    <p class="alert alert-danger"><%= alert %></p>
    <% end %>

File: db/migrate/20180418091041_add_liked_and_comment_id_to_likes.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddLikedAndCommentIdToLikes < ActiveRecord::Migration[5.1]
+  def change
+    add_column :likes, :liked, :boolean
```
Comment (rewitt94) [inline db/migrate/20180418091041_add_liked_and_comment_id_to_likes.rb]: isn't this code from the likes two branch..??

**Review by rewitt94 — COMMENTED** [review]
Let's go

**Review by Reena-Koudi — APPROVED** [review]
Approved!

**Pull request merged**